### PR TITLE
Barchart: Add gdev dashboard for tooltips

### PR DESCRIPTION
**Pull request by drew08t**
(follow-up to https://github.com/grafana/grafana/pull/67872)

Add a gdev dashboard for testing Bar Chart tooltip, particularly field overrides (though this can be the home for any future tooltip testing).

- [x] Datalink field override (https://github.com/grafana/support-escalations/issues/5613)
- [x] Hidden tooltip field override (https://github.com/grafana/support-escalations/issues/5367)
- [ ] ~~Tooltip Crash when single (https://github.com/grafana/grafana/issues/63307)~~ Unable to replicate this issue, will follow-up outside this scope.

image

### CODE REVIEW

File: devenv/dev-dashboards/panel-barchart/barchart-tooltips.json [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+        "type": "barchart"
+      }
+    ],
+    "refresh": "",
+    "schemaVersion": 38,
+    "style": "dark",
+    "tags": [
+        "gdev",
+        "panel-tests",
+        "barchart"
```
Comment (leeoniya) [inline devenv/dev-dashboards/panel-barchart/barchart-tooltips.json:472]: let's maybe add `graph-ng` as well. it helps to pull up all panels that are uPlot-based and i go through them all when updating the uPlot dependency.

https://github.com/grafana/grafana/blob/dba555976bb23f58b0d68e17bbc7dce6a8a695af/devenv/dev-dashboards/panel-barchart/barchart-autosizing.json#L610

**Comment by nmarrs** [comment]
I think we can target 10.1.x milestone instead and avoid backporting 👍

**Comment by leeoniya** [comment]
these are tests for a fix that went into 10.0.x (https://github.com/grafana/grafana/pull/67872), so i think backporting is appropriate here.

**Review by nmarrs — APPROVED** [review]
🎈

**Review by nmarrs — APPROVED** [review]
🎈

**Pull request merged**